atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1890,8 +1890,23 @@ private void alterGcAndTombstoneInternal(
 
     public void cleanUpSchemaMutationLockTablesState() throws Exception {
         Set<TableReference> tables = lockTables.getAllLockTables();
```

Comment:
I'm an idiot.  I didn't realize we were dropping tables here and not truncating, which basically makes my point in that issue incorrect.  I still like the idea of writing the cleared value given the complexity around schema changes though.

Do we have to handle other failure cases that your run teh CLI for?  LIke multiple locks tables existing or something?